package.json

```
@@ -5,14 +5,17 @@
     "dev": "vite",
     "build": "vite build --mode client && vite build",
     "preview": "wrangler dev",
-    "deploy": "bun run build && wrangler deploy"
+    "deploy": "bun run build && wrangler deploy",
+    "check": "biome check ./app",
+    "check:write": "biome check --write ./app"
```

`vite.config.ts`などもチェックしてほしいので、`biome check .`の方が良いと思います

```suggestion
    "check": "biome check .",
    "check:write": "biome check --write ."
```